docs/help/FAQ/Ssh.rst

```diff
@@ -0,0 +1,181 @@
+How do I customize settings for SSH?
+------------------------------------
+
+By default, including the **ssh** module will include ``ssh::server`` and
+``ssh::client``, configured with reasonable defaults for the OS & environment:
+
+.. code-block:: puppet
+
+   include 'ssh'
+
+.. NOTE::
+
+   The examples below feature ``include 'ssh::server'`` and ``include
+   'ssh::client'``, but most SIMP scenarios already include them both via
+   ``ssh``
+
+
+How do I manage settings for the SSH server?
+=======================================================
+
+Including `ssh::server` with the default options will manage the server with
+reasonable settings for each host's environment.
+
+
+Configuring ``ssh::server::conf`` from Hiera
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To customize the SSH server, edit the parameters of `ssh::server::conf` using
+Hiera or ENC (Automatic Parameter Lookup).
```

Comment:
ENC = external node classifier, so I don't understand why 'Automatic Parameter Lookup' is in parens aftewards.

Reply:
double back ticks needed.  Currently, ssh::server::conf renders in italics